src/app/screens/Discover/websites.json

```diff
@@ -162,6 +110,81 @@
       }
     ]
   },
+  {
+    "title": "trading",
+    "items": [
+      {
+        "title": "LNMarkets",
+        "subtitle": "Instant Bitcoin derivatives trading",
+        "logo": "https://cdn.getalby-assets.com/alby-extension-website-screen/lnmarkets.png",
+        "url": "https://lnmarkets.com/"
+      },
+      {
+        "title": "Kollider",
+        "subtitle": "Instant Bitcoin derivatives trading",
+        "logo": "https://cdn.getalby-assets.com/alby-extension-website-screen/kollider_thumbnail.png",
+        "url": "https://kollider.xyz/"
+      },
+      {
+        "title": "LOFT",
+        "subtitle": "Trade investment products in Bitcoin",
+        "logo": "https://cdn.getalby-assets.com/alby-extension-website-screen/loft-trade_thumbnail.png",
+        "url": "https://loft.trade/"
+      }
+    ]
+  },
+  {
+    "title": "gaming",
+    "items": [
+      {
+        "title": "Lightning Poker",
+        "subtitle": "Play Poker with Bitcoin",
+        "logo": "https://cdn.getalby-assets.com/alby-extension-website-screen/lightning-poker_thumbnail.png",
+        "url": "https://lightning-poker.com/"
+      },
+      {
+        "title": "LNBlackJack",
+        "subtitle": "Play Blackjack with Bitcoin",
+        "logo": "https://cdn.getalby-assets.com/alby-extension-website-screen/lnblackjack_thumbnail.png",
+        "url": "https://www.lnblackjack.com/"
+      },
+      {
+        "title": "Lightning Roulette",
+        "subtitle": "Play Roulette with Bitcoin",
+        "logo": "https://cdn.getalby-assets.com/alby-extension-website-screen/lightning-roulette_thumbnail.svg",
+        "url": "https://lightning-roulette.com/"
+      },
+      {
+        "title": "LNflip",
+        "subtitle": "Play coin flip matches against other players",
+        "logo": "https://cdn.getalby-assets.com/alby-extension-website-screen/lnflip_thumbnail.png",
+        "url": "https://www.lnflip.com/"
+      }
+    ]
+  },
+  {
+    "title": "nodeManagement",
+    "items": [
+      {
+        "title": "channel.ninja",
+        "subtitle": "Get recommended channel partners for your lightning node",
+        "logo": "https://cdn.getalby-assets.com/alby-extension-website-screen/channelninja.png",
+        "url": "https://channel.ninja/"
+      },
+      {
+        "title": "Amboss",
+        "subtitle": "Lightning Network node explorer",
+        "logo": "https://cdn.getalby-assets.com/alby-extension-website-screen/amboss-space_thumbnail.png",
+        "url": "https://amboss.space/"
+      },
+      {
+        "title": "LNnodeinsight.com",
+        "subtitle": "Develop your own data-driven Lightning Network insight",
+        "logo": "https://lnnodeinsight.com/www/LNnodeinsight-tinylogonobkgrnd-white.svg",
```

Comment:
@MoritzKa Can you please upload the logos files to our CDN and replace the paths here?

Reply:
Try this:
https://cdn.getalby-assets.com/alby-extension-website-screen/LNnodeinsight-tinylogonobkgrnd.svg

Reply:
Done, Here's how it looks :
![image](https://user-images.githubusercontent.com/65951872/226714188-4e945314-d9e6-4860-b3f2-e79f5bc6eab4.png)